aw_server/custom_watcher_pages.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from flask import Blueprint, send_from_directory, jsonify, request, current_app, escape
+from iso8601 import iso8601
+
+
+def get_bucket_name_from_watcher_name(buckets, watcher_name: str):
+    for bucket in buckets:
+        if bucket.startswith(watcher_name):
+            return bucket
+
+    logging.warning(f"Cannot find bucket for watcher {watcher_name}")
+    return None
+
+
+def get_custom_watcher_blueprint(custom_watcher_static_directories):
+    custom_watcher_blueprint = Blueprint("custom_watcher", __name__, url_prefix="/watcher")
+
+    @custom_watcher_blueprint.route("api/supported_watchers")
+    def custom_watcher_api_supported_pages():
+        return jsonify(list(custom_watcher_static_directories.keys()))
```

Comment:
Instead of having a `supported_watchers` endpoint, make the plain `pages/` endpoint (without a `<string:name>` passed) return the available custom static folders.

Reply:
That makes sense.